src/accessory.ts

```diff
@@ -111,13 +112,14 @@ export class VieramaticPlatformAccessory {
 
     this.service.setCharacteristic(
       this.Characteristic.Name,
-      device.specs.friendlyName
+      // device.specs.friendlyName <- Not too sure what the difference is between Name and ConfiguredName
+      accessory.displayName
     );
```

Comment:
changes here and for the remaining services shouldn't be needed anymore if we ias i suggested bellow inject override in `device.specs`

Reply:
Reversed all back to "device.specs.friendlyName" to ensure code is in same format as previously was.